src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1124,11 +1125,18 @@ Update Update( List<WithItem> with ):
     List<OrderByElement> orderByElements;
     boolean useColumnsBrackets = false;
     List<SelectExpressionItem> returning = null;
+    Token tk = null;
+    UpdateModifierPriority modifierPriority = null;
+    boolean modifierIgnore = false;
 }
 {
-   <K_UPDATE> { update.setOracleHint(getOracleHint()); } table=TableWithAlias()  
-        startJoins=JoinsList()
-
+   <K_UPDATE> { update.setOracleHint(getOracleHint()); }
+    [(tk = <K_LOW_PRIORITY>)
```

Comment:
Pardon me, I do not understand that line: would the token not always be "LOW_PRIORITY" and so modifierPriority = UpdateModifierPriority.LOW_PRIORITY?

Why would we not write this as
```
[ <K_LOW_PRIORITY> { modifierPriority = UpdateModifierPriority.LOW_PRIORITY; } ]
```

Am I missing something please?

Reply:
I fix that ok. I copy the logic from INSERT priority but they have more values...

Reply:
My fault sry ^^. Do I need something in this PR regarding your comment here ? https://github.com/JSQLParser/JSqlParser/pull/1382#issuecomment-956230110

Reply:
> Do I need something in this PR regarding your comment here ? [#1382 (comment)](https://github.com/JSQLParser/JSqlParser/pull/1382#issuecomment-956230110)

No, you did good.
I just used your contribution as an example, why we need to work on the Reserved Keyword Management in general and it is not certain, if @wumpz  will like my idea. If he does, then your new token `QUICK` will be handled automatically. If he does not, we can add it any time later to the Allowed Keywords.

Managing `Reserved Keywords` in a JavaCC Grammar is a bit weird and not straight forward.

Reply:
Ok no problem, thanks for the review anyway.